### DISCUSSION TITLE
Trip Map shows location not in Drives

### DISCUSSION THREAD

**Discussion by Ryan-Rogue** [teslamate-org/teslamate#2990] [Q&A] [answered]
I had the computer in my Model 3 replaced yesterday by Tesla and it appears that the default location on a new computer is the Tesla HQ. On my trip report in Grafana the map shows the red line teleporting out to Palo Alto and then back to the Salt Lake City service center I assume once the GPS updated. But none of the individual drives on that day contain this "trip" from CA to UT so i don't know which one to delete following the instructions from that. Is there any way to narrow it down so i don't just start blasting out drives. I kinda like seeing the little things Tesla did with the car that day.

![image](https://user-images.githubusercontent.com/59209856/202809807-bfdbe6f6-1929-4d4d-bf33-1e97933e11ce.png)

and the three drives that day (I wont show my home one, but California isn't part of it either, i promise)
![image](https://user-images.githubusercontent.com/59209856/202809936-50281afd-ad5c-47fb-9fcf-62173ffdb6fb.png)
![image](https://user-images.githubusercontent.com/59209856/202809959-49fcac78-2068-4d52-98ae-327c40f010ce.png)
![image](https://user-images.githubusercontent.com/59209856/202809988-c471e4dd-be4a-4718-ad46-c7189614b2d6.png)

**Comment by cwanja** [comment, marked as answer]
Check the positions table in the database and drop the position ID with the coordinates from Fremont.

**Reply by Ryan-Rogue** [reply]
Thanks, that's what I need to know! found 6 entries in the Positions table that had a lat long in California, removed them and that cleaned everything up!
